resources/js/Components/AvailabilitiesModal.vue

```diff
@@ -0,0 +1,288 @@
+            </template>
+          </tr>
+          <tr>
+            <td class="lt-grey p-3">Closing Hours</td>
+            <template v-for="availability in availabilities">
+              <td class="lt-grey p-3">{{ availability.closing_hours }}</td>
+            </template>
+          </tr>
+        </table>
+
+        <div class="flex justify-evenly pl-2 pr-2 pt-5 pb-5">
+          <div>
+            <h2>Legend</h2>
+            <p>
+              <em class="fas fa-circle fill-current text-green-600"></em>
+              Available
+            </p>
+            <p>
+              <em class="fas fa-circle fill-current text-red-600"></em>
+              Non-Available
+            </p>
+            <p>
+              <em class="fas fa-circle fill-current text-blue-600"></em>
+              Selected
+            </p>
+            <p><em class="fas fa-circle"></em> Blackbox</p>
+          </div>
+          <div>
+            <Calendar
+              :attributes="attributes"
+              @dayclick="onDayClick"
+              trim-weeks
+            />
+          </div>
+        </div>
+        <hr />
+
+        <div class="flex justify-evenly pl-2 pr-2 pt-5">
+          <div>
+            <h2>Bookings for selected day :</h2>
+            <template v-for="day in days">
+              <p>{{ day.date }}</p>
+            </template>
+          </div>
+          <div>
+            <div class="bg-yellow-400 p-3 m-1" v-for="reservation in reservations">
+              <p>
+                {{ "Start time: " + moment(reservation.start_time) }}
+              </p>
+              <p>
+                {{ "End time: " + moment(reservation.end_time) }}
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </template>
+  </jet-dialog-modal>
+</template>
+
+<script>
+import JetButton from "@src/Jetstream/Button";
+import JetInput from "@src/Jetstream/Input";
+import JetActionMessage from "@src/Jetstream/ActionMessage";
+import JetFormSection from "@src/Jetstream/FormSection";
+import JetInputError from "@src/Jetstream/InputError";
+import JetLabel from "@src/Jetstream/Label";
+import JetDialogModal from "@src/Jetstream/DialogModal";
+import JetDropdown from "@src/Jetstream/Dropdown";
+import JetDropdownLink from "@src/Jetstream/DropdownLink";
+import JetNavLink from "@src/Components/Navbar/NavLink";
+import JetSecondaryButton from "@src/Jetstream/SecondaryButton";
+import DialogModal from "@src/Jetstream/DialogModal";
+import Calendar from "v-calendar/lib/components/calendar.umd";
+import DatePicker from "v-calendar/lib/components/date-picker.umd";
+import moment from "moment";
+import axios from "axios";
+
+export default {
+  components: {
+    DialogModal,
+    JetButton,
+    JetInput,
+    JetFormSection,
+    JetActionMessage,
+    JetInputError,
+    JetLabel,
+    JetDropdown,
+    JetDropdownLink,
+    JetNavLink,
+    JetDialogModal,
+    JetSecondaryButton,
+    Calendar,
+    DatePicker,
+    moment
```

Comment:
Not sure if you need to declare moment as a component

Reply:
```suggestion
```